package.json

```diff
@@ -21,14 +21,15 @@
     "exchanges"
   ],
   "scripts": {
+    "prepare-core": "node ./scripts/prepare-core.js",
     "clean": "rimraf ./dist ./node_modules/.cache",
     "build": "rollup -c rollup.config.js",
     "watch": "rollup -w -c rollup.config.js",
     "check": "tsc --noEmit",
     "test": "jest",
     "coverage": "jest --coverage",
     "lint": "eslint . --ext .ts,.tsx",
-    "prepublishOnly": "run-s clean test build",
+    "prepublishOnly": "run-s pepare-core clean test build",
```

Comment:
Sorry another not 😅 let's add the core folder to the rimraf clean task and Run prepare after?

There's also a small typo in the tasks name here